ctapipe_io_magic/__init__.py

```diff
@@ -620,20 +646,30 @@ def prepare_subarray_info(self):
             2: [-34.99, 24.02, 0.00] * u.m
         }
 
-        if self.focal_length_choice == 'effective':
-            # Use the effective focal length that the coma aberration is corrected
-            focal_length = u.Quantity(17*1.0713, u.m)
-        else:
-            focal_length = u.Quantity(17, u.m)
+        equivalent_focal_length = u.Quantity(16.97, u.m)
+        effective_focal_length = u.Quantity(17*1.0713, u.m)
```

Comment:
not that it really matters, but was changing only the equivalent focal length to 16.97m, but keeping the base of 17m times the coma factor of 1.0713 intended? I am not sure myself which one would be better here 17  * 1.0713 or 16.97 * 1.0713

Reply:
For LSTProd2 simulations, the value adopted by EventSource in case of "nominal" focal length is 17.00, so maybe we should use 17, so that, if we want to use the equivalent focal length, both Monte Carlo and MAGIC data will have  consistent optics. This way, 17*1.0713=18.212, which is the MC effective focal length (in simtel logs)

Reply:
Indeed, it seems this is the case. Just for reference:

```python
In [1]: from eventio.simtel.simtelfile import SimTelFile

In [2]: simtelf.telescope_descriptions[2]["camera_settings"]["focal_length"]
Out[2]: 17.0

In [3]: simtelf.telescope_descriptions[2]["camera_settings"]["effective_focal_length"]
Out[3]: 18.211999893188477
```

Since even for MAGIC-only analysis those MC would be used, then there should be any problem. But even if for some reason one uses MAGIC MCs, the difference would be small. I will change tomorrow the `equivalent_focal_length` back to 17m if there are no objections.

Reply:
Discussing with @jsitarek , we came to the conclusion that we should use the updated value for the `equivalent_focal_length`, therefore 16.97m. I summarized the discussion in https://github.com/cta-observatory/magic-cta-pipe/discussions/140